### PR TITLE
clouddata: sync latest SP2 admin kiwi image

### PR DESCRIPTION
**Pull request by bmwiedemann**
later there needs to be some way to test and copy this image
to do staging on SLE changes

### CODE REVIEW

**Comment by mjura** [comment]
+1

File: scripts/clouddata/syncSLErepos [hunk 1]
```diff
@@ -207,4 +207,7 @@ for version in 6 7; do
     done
 done
 
+# sync admin images
+$rsync -L /mnt/dist/ibs/Devel:/Cloud:/Images/images_SLE_12_SP2/mkcloud-admin-node-SLE12SP2.x86_64.qcow2 /srv/nfs/images/x86_64/SLES12-SP2-untested.qcow2
```
Comment (dirkmueller) [inline scripts/clouddata/syncSLErepos]: can you do the same for aarch64 and s390x?

Comment (toabctl) [inline scripts/clouddata/syncSLErepos]: and also for the manila-service image?

**Comment by bmwiedemann** [comment]
updated to also sync arm and Z. I prefer that we add manila later, once we figure out the gating (to copy the -untested file to the one used by all tests).

**Comment by toabctl** [comment]
+1

**Comment by dirkmueller** [comment]
+1

**Pull request merged**